src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+	public String value;
+}
```

Comment:
This parent class can be better used to maintain duplicated codes from the child classes.
Examples:
-Common variables to be implemented here instead 
- Using a parent constructor to assign. E.g. super(value)